Tasty/Tasty/Domain/TastyRecipes/Scenes/RecipeDetailsScreen/RecipeDetailsViewController.swift

```diff
@@ -0,0 +1,216 @@
+        static let backgroundColorName = "BackgroundColor"
+        static let loadingText = "Loading..."
+        static let marshColor = "MarshColor"
+        static let tinderAppText = "Tinder App"
+        static let favoriteImageName = "favorite"
+        static let separatorText = "\n"
+    }
+    
+    // MARK: - Private Visual Components
+    
+    private let foodImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.layer.cornerRadius = 20
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.layer.masksToBounds = true
+        imageView.clipsToBounds = true
+        imageView.contentMode = .scaleAspectFill
+        imageView.backgroundColor = UIColor(named: Constants.marshColor)
+        return imageView
+    }()
+    
+    private lazy var showTinderButton: UIButton = {
+        let btn = UIButton()
+        btn.translatesAutoresizingMaskIntoConstraints = false
+        btn.addTarget(self, action: #selector(showTinderButtonAction), for: .touchUpInside)
+        btn.backgroundColor = UIColor(named: Constants.marshColor)
+        btn.setTitle(Constants.tinderAppText, for: .normal)
+        btn.titleLabel?.font = UIFont.boldSystemFont(ofSize: 30)
+        btn.setTitleColor(.white, for: .normal)
+        btn.layer.cornerRadius = 15
+        return btn
+    }()
+    
+    private lazy var favoritesButton: UIButton = {
+        let btn = UIButton()
+        let originalImage = UIImage(named: Constants.favoriteImageName)
+        let tintedImage = originalImage?.withRenderingMode(.alwaysTemplate)
+        btn.translatesAutoresizingMaskIntoConstraints = false
+        btn.setImage (tintedImage, for: .normal)
+        btn.tintColor = UIColor(named: Constants.marshColor)
+        btn.addTarget(self, action: #selector(favoritesButtonAction), for: .touchUpInside)
+        return btn
+    }()
+    
+    private let recipeTitleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.boldSystemFont(ofSize: 30.0)
+        label.textColor = UIColor(named: Constants.marshColor)
+        label.textAlignment = .center
+        return label
+    }()
+    
+    private let compoundLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.boldSystemFont(ofSize: 22)
+        label.numberOfLines = 0
+        label.textColor = UIColor(named: Constants.marshColor)
+        label.textAlignment = .center
+        return label
+    }()
+    
+    // MARK: - Public Properties
+    
+    var interactor: RecipeDetailsBusinessLogic?
+    var router: RecipeDetailsRoutingLogic?
+    
+    // MARK: Initializers
+    
+    init(interactor: RecipeDetailsBusinessLogic?, router: RecipeDetailsRoutingLogic) {
+        super.init(nibName: nil, bundle: nil)
+        self.interactor = interactor
+        self.router = router
```

Comment:
```suggestion
        self.interactor = interactor
        self.router = router
        super.init(nibName: nil, bundle: nil)
```
super.init вызывается после инициализации всех свойств. Напиши мне если тут не так